pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -2674,6 +2674,21 @@ in
     ];
   };
 
+  parrot-nvim = super.parrot-nvim.overrideAttrs {
+    runtimeDeps = [
+      curl
+    ];
```

Just a style suggestion
```suggestion
    runtimeDeps = [ curl ];
```